app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +54,15 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $this->reportbackService->updateReportbackItems($request->all());
```

Comment:
(sorry for the out of order comments) This should receive a return value from `updateReportbackItems()` and then it should use a Transformer to format the response to the client. 

A Transformer is a class that allows you to format the response that an API returns to a client using the API in a way that makes it more usable. For example, when returning a date from the database, you might want that date to show in the API response formatted in a readable way. I am sorta confused about how this endpoint is retuning any formatted response but it seems like it is? The reason why I suggest using a Transformer is because it is best practice and allows use to follow the separation of concerns pattern.

I already created a `ReportbackItemTransformer` [class](https://github.com/DoSomething/rogue/blob/master/app/Http/Transformers/ReportbackItemTransformer.php) that I think you can use here. This class basically just defines what the API response should look like from the rogue api when returning `ReportbackItem` models.

I think the steps for doing this should be pretty straight forward: 

In the `ReportbackController` class you pretty much follow the same thing that is being done for reportbacks except: 

1) Create a new `protected` variable like `$itemTransformer` 
2) Set that in the constructor
3) add a `return` statement in this function that does something like `return $this->item($item, $code, $itemTransformer);`
4) when you are returning a collection of items you will probably have to do something like `return $this->collection($items, $code, $itemTransformer);

I hope that makes sense! I can swing by if you have any questions.